docs/themes/avocadocs/layouts/_default/single.html

```diff
@@ -18,7 +18,7 @@ <h1>{{ .Title }}</h1>
           <div class="content-grid">
 
             <!-- Table of contents -->
-            {{ if ne .Params.toc false }}
+            {{ if and (ne .Params.toc false) (ne .TableOfContents "<nav id=\"TableOfContents\"></nav>") }}
```

Comment:
🐞
Could this change have broken [this page](https://pr-2054-m3jye5y-652soceglkw4u.eu-3.platformsh.site/integrations/observability.html)? compared to [its live counterpart](https://docs.platform.sh/integrations/observability.html)?